README.md

```diff
@@ -3,9 +3,9 @@
 [![Gem Version](https://badge.fury.io/rb/infopark_webcrm_sdk.svg)](http://badge.fury.io/rb/infopark_webcrm_sdk)
 [![Build Status](https://travis-ci.org/infopark/webcrm_sdk.svg?branch=master)](https://travis-ci.org/infopark/webcrm_sdk)
```

Comment:
We should remove this "Build Status" line since Travis CI no longer works, at least we're no longer testing with Travis CI.